src/Recorder/FilesystemRecorder.php

```diff
@@ -59,15 +65,19 @@ public function replay(string $name): ?ResponseInterface
 
         $this->log('Response replayed from {filename}', $context);
 
-        return Psr7\parse_response(file_get_contents($filename));
+        return Message::parseResponse(file_get_contents($filename));
```

Comment:
This change is [causing an error on the pipeline](https://travis-ci.com/github/php-http/vcr-plugin/jobs/445151660), so you can either keep the function or bumb `guzzlehttp/psr7` version.

Reply:
done